src/query/atomWithInfiniteQuery.ts

```diff
@@ -1,14 +1,13 @@
-import { InfiniteQueryObserver, isCancelledError } from '@tanstack/query-core'
 import type {
   InfiniteData,
   InfiniteQueryObserverOptions,
   QueryKey,
-  QueryObserverResult,
   RefetchOptions,
   RefetchQueryFilters,
 } from '@tanstack/query-core'
+import { atomsWithInfiniteQuery } from 'jotai-tanstack-query'
 import { atom } from 'jotai'
```

Comment:
Hi @dai-shi! Are there plans of importing `atom` from `valtio/vanilla` instead? Thanks!

Reply:
`jotai/query` will be deprecated soon.
So, you want to use `jotai-tanstack-query` and, in that case, we have a PR open.

https://github.com/jotai-labs/jotai-tanstack-query/pull/18

There, find codesandbox-ci comment and click `here`. You will then find "Local Install Instructions".